superset/views/superset_decorators.py

```diff
@@ -0,0 +1,15 @@
+import functools
+from flask import (
+    redirect, session
+)
+
+def check_target_url(f):
```

Comment:
should we rename to `redirect_to_last_url` or `navigate_to_user_url` or `redirect_to_target_url`.

We are not checking existence of url but also navigating to it

Reply:
rename to redirect_to_target_url